storage/innobase/log/log0recv.cc

```diff
@@ -1272,6 +1274,7 @@ inline uint32_t log_block_calc_checksum_format_0(const byte *b)
 ATTRIBUTE_COLD static dberr_t recv_log_recover_pre_10_2()
 {
   uint64_t max_no= 0;
+  dberr_t err;
```

Comment:
Please use code similar to `recv_log_recover_10_4()` here, and do not declare a local variable for the entire function scope.